File: dotnet/SimpleSauce/SimpleSauce/DataCenter.cs
namespace SimpleSauce
{
    public class DataCenter
    {
        private DataCenter(string value) { Value = value; }

        public string Value { get; set; }

        public static DataCenter UsWest { get { return new DataCenter("https://ondemand.saucelabs.com/wd/hub"); } }
    }
}

File: java/src/main/java/com/saucelabs/simplesauce/DataCenter.java
package com.saucelabs.simplesauce;

import lombok.Getter;

public enum DataCenter {
    US_WEST("ondemand.us-west-1.saucelabs.com"),
    US_EAST("ondemand.us-east-1.saucelabs.com"),
    EU_CENTRAL("ondemand.eu-central-1.saucelabs.com");

    @Getter private final String value;

    DataCenter(String value) {
        this.value = value;
    }

    public String toString() {
        return value;
    }
}
